Marlin/src/lcd/extui/dgus_reloaded/DGUSTxHandler.h

```diff
@@ -35,7 +35,7 @@ namespace DGUSTxHandler {
   #endif
 
   void PositionZ(DGUS_VP &);
-  void Ellapsed(DGUS_VP &);
+  void Elapsed(DGUS_VP &);
```

Comment:
"Ellapsed" is used elsewhere for LCD/dgus stuff, so this may break some things.

Reply:
my bad, this change wasn't supposed to be included in this PR. Will revert now

Reply:
7eadfe7